statvar_imports/oecd/quarterly_gdp/pvmap.csv

```diff
@@ -0,0 +1,36 @@
+key,property,value,property2,value2,property3,value3
+# PV map for OECD Quarterly GDP Growth Rate,,,,,,
+,,,,,,
+# Dimensions,,,,,,
+FREQ:Q,observationPeriod,P3M,,,,
```

Comment:
What about FREQ:A as some rows in sample_input.csv:26?

Reply:
Since this data import focuses on quarterly GDP values, I explicitly set the `download.sh` code to only fetch `FREQ:Q` data (and consequently ignore annual data). That's why we don't need to mention `FREQ:A` in pvmap either as it won't encounter them.

Reply:
It's possible the `sample_input.csv` file you saw was an older version containing annual data. I just re-generated again to be sure that no annual data is present in it.